### DISCUSSION TITLE
R5.1.0.1 Sunbird ED sign off

### DISCUSSION THREAD

**Discussion by ShailandraOjha** [Sunbird-Ed/Community#372] [General Discussion]
Hi All,

This is the sign-off for R5.1.0.1 for Sunbird-ED as We have completed the new feature verification and impacted Regression testing on the Sunbird-ED environment, Please find the detailed test case execution sheet [[Here](https://docs.google.com/spreadsheets/d/1EqJTBOmDjl3JYWQF3FCnT1YyC29MJwyMTJg-rHn0-2A/edit#gid=1581973510)]

Environment: [Sunbird-ED](url)
Testing completion percentage: 100% Completed
Execution Sheet: [Link](https://docs.google.com/spreadsheets/d/1EqJTBOmDjl3JYWQF3FCnT1YyC29MJwyMTJg-rHn0-2A/edit#gid=1581973510)
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10614)

![image](https://user-images.githubusercontent.com/44188853/215094758-784827a1-cdad-4ce9-bc50-3943fd9db057.png)

​Also please find the Regression + New feature sign-off sheets from all the circles [[here](https://docs.google.com/spreadsheets/d/1fpPBK-ElS-itAXKyNHkgBh1vr_le4cleUB6OYvpsAVo/edit#gid=499496010)]

![image](https://user-images.githubusercontent.com/44188853/215094623-aec3d553-3523-467e-ab35-c6672f3f3e2a.png)

@rajeevsathish @surendrasinghs @sharathkashyap @rhwarrier @anisha147 @luckynvdu @vinukumar-vs(https://github.com/vinukumar-vs) [@pallakartheekreddy](https://github.com/pallakartheekreddy) @reshmi-nair @anandp504 @gandham-santhosh@kameshbhr @mmanimegalai @madhucr @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @ashokreddy1208 @mmanimegalai @chitranshu-keshav [AdarshP29](https://github.com/AdarshP29)@Ak0094 @anandvarada @Aparna-karri591 @Nadiya496 @Naveenkumar739 @PoornimaRajesh @Priyakashi @manju4201 @swayangjit @tarang27 @anunagaraj https://github.com/ashokreddy1208

Regards,
Shailandra Ojha

**Comment by ashokreddy1208** [comment]
@keshavprasadms @gandham-santhosh @G33tha @vinukumar-vs @santhosh-tg @saiakhil46 please provide deployment tracker for this release as we need to take this to Diksha as part of 5.1 release.

cc @ anandvarada @rajeevsathish @Krishnaj20

**Comment by anandvarada** [comment]
@vinukumar-vs @gandham-santhosh @Krishnaj20  Do share the tracker for this hot fix ..

**Reply by gandham-santhosh** [reply]
@ashokreddy1208 Managed Lern TAG's are updated at the upgrading section of microsite https://ed.sunbird.org/use/installing-ed-platform/release-5.1.0-draft/build . Please refer the last 3 rows of "**Build and Deploy for Sunbird**" section

**Comment by ashokreddy1208** [comment]
Thanks @gandham-santhosh, We also need mobile app tag for Managed Lern
